migration.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python3
+
+import argparse
+import json
+
+import requests
+
+
+def prepare_labels(rule):
+    if "labels" not in rule:
+        return
+
+    labels = rule["labels"]
+    labels.update({
+        "percona_alerting": "1",
+        "severity": rule["severity"].lstrip("SEVERITY_").lower(),
+        "template_name": rule["template_name"]
+    })
+    return labels
+
+
+def prepare_annotations(rule):
+    annotations = rule["annotations"]
+    annotations.update({"rule": rule["name"]})
+
+    return annotations
+
+
+def prepare_expression(rule):
+    expr = rule["expr"]
+
+    if "filters" not in rule:
+        return expr
+
+    for f in rule["filters"]:
```

Comment:
```
for f in rule.get('filters', []):
  key = f.get('key', "")
  value = f.get('key', '')
  expr = f"label_match({expr}, "{key}", "{value}")
```
This is more pythonic way and it has some fallbacks to the empty data type without any exceptions